.storybook/main.js

```diff
@@ -78,4 +78,5 @@ module.exports = {
 
     return config;
   },
+  staticDirs: ['./static', { from: '../data', to: 'data' }],
```

Comment:
This serves Simorgh fixture data from the same place as the static data within Storybook: https://github.com/bbc/simorgh/tree/latest/.storybook/static/data